userspace/libscap/scap_savefile.h

```diff
@@ -52,7 +52,7 @@ typedef struct _block_header
 // Must be increased every time we change the capture format
 // (e.g. most of the changes in the event table, like adding
 // a syscall)
-#define CURRENT_MINOR_VERSION	2
+#define CURRENT_MINOR_VERSION	3
```

Comment:
@mattpag, do we need a version bump here?

Reply:
No we don't. That comment should have been removed when we introduced the flexible captures support with https://github.com/draios/sysdig/pull/1163. I'll make a PR to fix it.

Reply:
Ok thanks will revert!